src/style.css

```diff
@@ -0,0 +1,112 @@
+/* stylelint-disable string-quotes */
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-bold";
+  font-style: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+form {
+  margin: 20px 0;
+}
+
+.searh-bar-big {
+  font-family: Avenir;
+  width: 100%;
+  height: 70px;
+  font-weight: 300px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding: 0 62px;
+}
+
+.searh-bar-big::placeholder {
+  font-family: Avenir;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.searh-bar-big:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searh-bar-big:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-family: Avenir-bold;
+  font-weight: 300;
+}
+
+.searh-bar-small {
+  font-family: Avenir;
+  width: 100%;
+  height: 42px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding: 0 33px;
+}
+
+.searh-bar-small:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searh-bar-small:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-family: Avenir-bold;
+}
+
+.searh-bar-small::placeholder {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
+
+.label-class-big {
+  position: relative;
+}
+
+.label-class-big::before {
+  content: "";
+  background-image: url('images/Search.svg');
+  background-repeat: no-repeat;
+  background-size: contain;
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 2px;
```

Comment:
try to use 0 value for top

Reply:
fixed

Reply:
not fixed 
![image](https://user-images.githubusercontent.com/70009205/232833661-be6cb9e9-18d6-4c25-944a-9bfeb71a8d4d.png)
BTW all your tests have failed, especially the font boldness and the small input icon are higher than necessary. Let's first set all the properties so that the tests pass (even if it is not zero here)

Reply:
Yes, I know that all the tests failed, I just don't understand what exactly I did wrong, because visually it's very close to what is needed, but still the tests failed. I can't figure out what to focus on...